projects/packages/forms/src/dashboard/class-dashboard.php

```diff
@@ -120,17 +121,35 @@ public function add_admin_submenu() {
 			return;
 		}
 
-		remove_menu_page( 'feedback' );
+		$is_wpcom = ( new Host() )->is_wpcom_simple();
 
-		add_menu_page(
-			__( 'Form Responses', 'jetpack-forms' ),
-			_x( 'Feedback', 'post type name shown in menu', 'jetpack-forms' ),
-			'edit_pages',
-			'jetpack-forms',
-			array( $this, 'render_dashboard' ),
-			'dashicons-feedback',
-			25 // Places 'Feedback' under 'Comments' in the menu
-		);
+		// MODERN VIEW -- remove the old submenu and add the new one.
+		// Check if Polldaddy/Crowdsignal plugin is active
+		if ( ! $is_wpcom && ! is_plugin_active( 'polldaddy/polldaddy.php' ) ) {
+			remove_menu_page( 'feedback' );
+
+			add_menu_page(
+				__( 'Form Responses', 'jetpack-forms' ),
+				_x( 'Feedback', 'post type name shown in menu', 'jetpack-forms' ),
+				'edit_pages',
+				'jetpack-forms',
+				array( $this, 'render_dashboard' ),
+				'dashicons-feedback',
+				25 // Places 'Feedback' under 'Comments' in the menu
+			);
+		} else {
+			remove_submenu_page( 'feedback', 'edit.php?post_type=feedback' );
+
+			add_submenu_page(
+				'feedback',
+				__( 'Form Responses', 'jetpack-forms' ),
+				_x( 'Form Responses', 'menu label for form responses', 'jetpack-forms' ),
+				'edit_pages',
+				'jetpack-forms',
+				array( $this, 'render_dashboard' ),
+				0 // as far top as we can go since responses are the default feedback page.
+			);
+		}
```

Comment:
Can you give a short explanation on the rational for this this if/else?
- if self hosted without Crowdsignal: add **menu** page
- if wpcom or Crowdsignal: add a **submenu** page

Reply:
Yep! Pay close attention to `add_menu_page` vs `add_submenu_page`.

If polldaddy is not present:
- we can mess with the menu, so...
- remove the MENU we created with the plugin, slug `feedback`, since it's not the one we want
- add new MENU entry for Forms dashboard (this only reasonable since we're sure we're not sharing the menu with other entries)

Otherwise:
- we need to use/keep the MENU page
- remove only the SUBMENU, slug `edit.php?post_type=feedback`
- create the new SUBMENU for `jetpack-forms`
- since we're inside the Feedback menu and we just created a new one, place it as far top as we can (`0`)